UnityProject/Assets/Scripts/Messages/Client/Interaction/RequestExamineMessage.cs

```diff
@@ -43,16 +43,19 @@ public override IEnumerator Process()
 		// Look for examinables.
 		var examinables = NetworkObject.GetComponents<IExaminable>();
 		string msg = "";
-		foreach (var examinable in examinables)
-		{
-			msg += $"{examinable.Examine()}\n";
-		}
+		IExaminable examinable;
 
-		if (msg.Length > 0)
+		for (int i = 0; i < examinables.Count(); i++) 
 		{
-			msg = msg.Substring(0, msg.Length-2);
+			examinable = examinables[i];
+			msg += $"{examinable.Examine()}";
```

Comment:
You don’t need to do string interpolation and just add examinable.Examine() without quotes, but that’s a minor nitpick. Also I’ve got a feeling that this whole loop could be avoided by simply using string.Join...